molgenis-app-manager/src/test/java/org/molgenis/app/manager/controller/AppControllerTest.java

```diff
@@ -140,28 +179,90 @@ public void testServeAppInactiveApp() throws Exception
 
 		AppResponse appResponse = AppResponse.create(app);
 		when(appManagerService.getAppByUri("uri")).thenReturn(appResponse);
-		String expectedMessage = "";
-		try
-		{
-			mockMvc.perform(get(AppController.URI + "/uri/")).andExpect(status().is4xxClientError());
-		}
-		catch (Exception e)
-		{
-			expectedMessage = e.getCause().getMessage();
-		}
-		finally
-		{
-			assertEquals(expectedMessage, "uri:uri");
-		}
+
+		mockMvc.perform(get(AppController.URI + "/uri/"))
+			   .andExpect(status().is4xxClientError())
+			   .andExpect(model().attribute("errorMessageResponse",
+					   ErrorMessageResponse.create("Access denied for inactive app at location /app/uri", "AM07")))
```

Comment:
I'd expect the style to match other exception messages, i.e.
> Access denied for inactive app at location '/app/uri'.

(But the PR doesn't contain this message.)